src/components/Tags/index.tsx

```diff
@@ -232,7 +232,7 @@ const Tags = ({
     >
       {tagsState.map(tag => (
         <Tag
-          variant={variant === 'no-border' ? 'base' : variant}
```

Comment:
So we loose variant on Tags ?

Reply:
Probably we will have component review for `Tags`,  I saw this with @matthprost, and It's fine to have one variant for tag.
checking on shire, we don't use any variant for tags just `no-border` in a couple of them